DOCUMENTATION.md

```diff
@@ -0,0 +1,101 @@
+# Documentation Guide
+
+M3 is a large and complex project, and any help you can offer to explain it better is most welcome.
+
+## Setup Documentation Locally
+
+### Prerequisites
+
+The M3 documentation uses [Hugo](https://gohugo.io/), a static site generator written in Go. [Find installation instructions in the Hugo documentation](https://gohugo.io/getting-started/installing/).
+
+If you also want to run and serve the M3 documentation, you need the custom theme, [Victor](https://github.com/chronosphereio/victor). Clone or download it into the same directory as the M3 codebase.
+
+To run some tests and test the production build steps, you need [Docker](https://www.docker.com).
```

Comment:
Are there any memory / capacity reqs as well? For example, when running my M3 workshop with the Docker compose yaml I needed to increase the desktop capacity settings in Docker.

Reply:
@gibbscullen For these tests, nope, it's just running Hugo in a container, not M3 itself. Do you think I should mention that?

Reply:
Hmm, wouldn't hurt to